sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/AbstractReactiveDatabaseIT.java

```diff
@@ -0,0 +1,243 @@
+    }
+
+    @Test
+    public void testNamedSearch() {
+        Response response = given()
+                .when().get("/hibernate/books/starts_with/Thinking");
+        String result = response.getBody().asString();
+        Assertions.assertTrue(result.contains("Thinking fast and slow"));
+    }
+
+    @Test
+    @Order(3)
+    public void testAuthorCreation() {
+        Response post = given()
+                .contentType(ContentType.JSON)
+                .post("/hibernate/author/create/Plato");
+        Assertions.assertEquals(HttpStatus.SC_CREATED, post.statusCode());
+        String result = given()
+                .when().get("/library/author/7")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("Plato", result);
+    }
+
+    @Test
+    public void testAuthor() {
+        Response response = given()
+                .when().get("/library/author/2");
+        String result = response.then()
+                .extract().body().asString();
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Assertions.assertEquals("Vern", result);
+    }
+
+    @Test
+    @Order(1)
+    public void testCreation() {
+        Response post = given()
+                .contentType(ContentType.JSON)
+                .post("/library/author/Wodehouse");
+        Assertions.assertEquals(HttpStatus.SC_CREATED, post.statusCode());
+        String result = given()
+                .when().get("/library/author/5")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("Wodehouse", result);
+    }
+
+    @Test
+    @Order(2)
+    public void testTooLongName() {
+        Response creation = given().contentType(ContentType.JSON).post("library/author/Subrahmanyakavi");
+        Assertions.assertEquals(HttpStatus.SC_BAD_REQUEST, creation.statusCode());
+        given()
+                .when().get("/library/author/6")
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND);
+    }
+
+    @Test
+    public void testGeneratedId() {
+        String author = given()
+                .when().get("/library/author/2")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("Vern", author);
+        Response creation = given().put("library/books/2/Around_the_World_in_Eighty_Days");
+        Assertions.assertEquals(HttpStatus.SC_CREATED, creation.statusCode());
+        String result = given()
+                .when().get("library/books/author/Vern")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("[Around_the_World_in_Eighty_Days]", result);
+    }
+
+    @Test
+    public void deletion() {
```

Comment:
Please take a look at test names. Try to make uniform naming. The same applies for test: dto, get*, set*, validationWithoutPanache

Reply:
Changed test names to the single style.